annlite/index.py

```diff
@@ -621,6 +620,30 @@ def snapshot_path(self):
             return paths[0]
         return None
 
+    @property
+    def remote_store(self):
+        client = hubble.Client(max_retries=None, jsonify=True)
```

Comment:
```
try:
    import hubble
except:
    raise Exception('Please instal `hubble-sdk` ......')
    
```

Reply:
The `jina-hubble-sdk`  will be installed with `jina` then.

Reply:
With `jina-hubble-sdk` not installed, `import jina` will not succeed.

Reply:
I see, we can remove the try catch block

Reply:
Fixed.